website/content/en/docs/best-practices/pod-security-standards.md

```diff
@@ -149,11 +221,175 @@ spec:
+If you are looking to use the metrics to do the checks then, ensure that you check its 
+results before and after performing the tests, see an example: 
+
+```sh
+kubectl get --raw /metrics | prom2json | jq '[.[] | select(.name=="pod_security_evaluations_total") ]'
+```
+```json
+[
+  {
+    "name": "pod_security_evaluations_total",
+    "help": "[ALPHA] Number of policy evaluations that occurred, not counting ignored or exempt requests.",
+    "type": "COUNTER",
+    "metrics": [
+      {
+        "labels": {
+          "decision": "allow",
+          "mode": "enforce",
+          "policy_level": "privileged",
+          "policy_version": "latest",
+          "request_operation": "create",
+          "resource": "pod",
+          "subresource": ""
+        },
+        "value": "29"
+      },
+      {
+        "labels": {
+          "decision": "allow",
+          "mode": "enforce",
+          "policy_level": "privileged",
+          "policy_version": "latest",
+          "request_operation": "update",
+          "resource": "pod",
+          "subresource": ""
+        },
+        "value": "0"
+      },
+      {
+        "labels": {
+          "decision": "deny",
+          "mode": "audit",
+          "policy_level": "restricted",
+          "policy_version": "latest",
+          "request_operation": "create",
+          "resource": "controller",
+          "subresource": ""
+        },
+        "value": "6"
+      },
+      {
+        "labels": {
+          "decision": "deny",
+          "mode": "audit",
+          "policy_level": "restricted",
+          "policy_version": "latest",
+          "request_operation": "create",
+          "resource": "pod",
+          "subresource": ""
+        },
+        "value": "7"
+      },
+      {
+        "labels": {
+          "decision": "deny",
+          "mode": "warn",
+          "policy_level": "restricted",
+          "policy_version": "latest",
+          "request_operation": "create",
+          "resource": "controller",
+          "subresource": ""
+        },
+        "value": "6"
+      },
+      {
+        "labels": {
+          "decision": "deny",
+          "mode": "warn",
+          "policy_level": "restricted",
+          "policy_version": "latest",
+          "request_operation": "create",
+          "resource": "pod",
+          "subresource": ""
+        },
+        "value": "7"
+      }
+    ]
+  }
+]
+```
+Also, you might be able to create Prometheus alerts such as:
+
+```sh
+sum (increase(pod_security_evaluations_total{decision="deny",mode="audit"}[1h])) by (policy_level)
+```
+
+### (Recommended) Now, how could I automate this checks (e2e) to ensure that my solutions can run under the policies?
+
+In the tests, you can create and label the namespace with the desired policy (i.e. restricted). So that you can verify if you 
+will check warnings and/or if the Pod/Containers are `Running` successfully. It can be something like:
```

Comment:
Nit:
```suggestion
In the tests, you can create and label a namespace with the desired policy (i.e. restricted). This would allow you to verify
if any warnings/errors occur and if the Pod(s)/Container(s) are in the `Running` state. It can be something like:
```